### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by cjsfj** [pikselpalette/fluent-plugin-flat-json-parser#3]
Seems that when I try to install, this is showing up as not found. Is it still maintained?

`# gem install fluent-plugin-flat-json-parser
ERROR:  Could not find a valid gem 'fluent-plugin-flat-json-parser' (>= 0) in any repository`

**Comment by craiggenner** [comment]
I don't believe this was ever pushed to a repo. It was built locally into a deb package and installed that way.

**Comment by cjsfj** [comment]
Thanks Craig. i'll try building it locally.
